CoreArrangementPlugins/release/modules/ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/uncollide-arrangement.md

```diff
@@ -0,0 +1,46 @@
+# Uncollide
+
+<table class="table table-striped">
+<thead>
+<tr class="header">
+<th>Constellation Action</th>
+<th>User Action</th>
+<th style="text-align: center;">Menu Icon</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>Run Uncollide</td>
+<td>Arrange -&gt; Uncollide</td>
+<td style="text-align: center;"><img src="../ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/resources/uncollide.png" alt="Uncollide Icon" /></td>
+</tr>
+
+<tr class="even">
+<td>Run Uncollide 3D</td>
+<td>Arrange -&gt; Uncollide 3D</td>
+<td style="text-align: center;"><img src="../ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/resources/uncollide3D.png" alt="Uncollide 3D Icon" /></td>
+</tr>
+</tbody>
+</table>
+
+
+Selected nodes are postioned such that they don't "collide" with one another. 
+If no nodes are selected, then all nodes on the graph will be affected. 
+
+Uncollide, contrast to Uncollide 3D, only operates on the X and Y axis.
```

Comment:
This could re-worded slightly (3D accounts for all 3 axes whereas 2D only accounts for X and Y)